db/seeds.rb

```diff
@@ -56,4 +56,6 @@ def create_ping_things(number)
 
   create_users
   create_ping_things(240)
-end
\ No newline at end of file
+
+  load(Rails.root.join('script', 'add_current_epoch.rb') )
+end
```

Comment:
Please also review other data that is created in seeds and can be removed, eg. there's no point in creating ping things in seeds. There are separate scripts for them. Also, code related to ping things in seed is not very good.

Reply:
It'd be nice to add puts statement to seeds that will tell the user where he should look for the scripts